keploy/keploy.go

```diff
@@ -340,26 +357,52 @@ func (k *Keploy) Test() {
 }
 
 func (k *Keploy) start(total int) (string, error) {
-	url := fmt.Sprintf("%s/regression/start?app=%s&total=%d&testCasePath=%s&mockPath=%s", k.cfg.Server.URL, k.cfg.App.Name, total, k.cfg.App.TestPath, k.cfg.App.MockPath)
-	body, err := k.newGet(url)
-	if err != nil {
-		return "", err
-	}
 	var resp map[string]string
+	if k.cfg.Server.GrpcEnabled{
+		res, err := k.grpcClient.Start(k.Ctx, &proto.StartRequest{
+			Total:        strconv.Itoa(total),
+			App:          k.cfg.App.Name,
+			TestCasePath: k.cfg.App.TestPath,
+			MockPath:     k.cfg.App.MockPath,
+		})
+		if err != nil {
+			return "", err
+		}
+		result, _ := res.Descriptor()
```

Comment:
Please add error handling to prevent panic in edge case.

Reply:
Done